sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/RoslynDataImporter.java

```diff
@@ -26,17 +26,22 @@
 import org.sonar.api.batch.ScannerSide;
 import org.sonar.api.batch.sensor.SensorContext;
 import org.sonar.api.rule.RuleKey;
+import org.sonar.api.utils.log.Logger;
+import org.sonar.api.utils.log.Loggers;
 import org.sonarsource.dotnet.shared.sarif.SarifParserCallback;
 import org.sonarsource.dotnet.shared.sarif.SarifParserFactory;
 
 @ScannerSide
 public class RoslynDataImporter {
-  public void importRoslynReport(List<Path> reportPaths, final SensorContext context, Map<String, List<RuleKey>> activeRoslynRulesByPartialRepoKey) {
+  private static final Logger LOG = Loggers.get(RoslynDataImporter.class);
+
+  public void importRoslynReports(List<Path> reportPaths, final SensorContext context, Map<String, List<RuleKey>> activeRoslynRulesByPartialRepoKey) {
     Map<String, String> repositoryKeyByRoslynRuleKey = getRepoKeyByRoslynRuleKey(activeRoslynRulesByPartialRepoKey);
     SarifParserCallback callback = new SarifParserCallbackImpl(context, repositoryKeyByRoslynRuleKey);
 
+    LOG.info("Importing Roslyn {} {}", reportPaths.size(), pluralize("report", reportPaths.size()));
```

Comment:
Message formatting: won't this return e.g. _Importing Roslyn 2 reports_?

Reply:
good catch